Personalized combat messages part 2

kingofkosmos opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Adds more "personalized" combat messages for all participants in a fight: the attacker, the victim and the spectators. Continuing to do what the previous PR did in #46388 (previous was mostly unarmed combat).

Old combat messages (attacker & spectator get the same message):
![old_scene](https://user-images.githubusercontent.com/5420151/89927372-e225c380-dc0e-11ea-951e-b8c4e84fff5e.png)

New combat messages (everyone gets their own message):
![new_scene](https://user-images.githubusercontent.com/5420151/90008909-c7525e00-dca5-11ea-9b95-02c2b009f5f9.png)


<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

PS. I'm not 100% sure how this should work in transforming.dm, _true_devil.dm, holo_effect.dm.

## Why It's Good For The Game
Easier to see what is happening and to whom.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
spellcheck: added personal versions for combat messages (e.g: "You whip John Smith!" instead of "Devin Spock whips John Smith!")
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

kingofkosmos committed:
* made combat messages "personalized" for the attacker, the victim and spectators.
* biggest changes in item_attack.dm, carbon_defense.dm and animal_defense.dm.
* divided all existing attack_verbs --> attack_verb_continuous and attack_verb_simple.
* (not 100% sure how this should work in transforming.dm, _true_devil.dm, holo_effect.dm.)

kingofkosmos committed:
* changes attack_verb time tenses from simple past to present continuous.

Rohesie commented on code/modules/mob/living/carbon/carbon_defense.dm:
```diff
@@ -99,9 +99,12 @@
-	var/message_verb = "attacked"
-	if(length(I.attack_verb))
-		message_verb = "[pick(I.attack_verb)]"
+	var/message_verb_continuous = "attacks"
+	var/message_verb_simple = "attack"
+	if(length(I.attack_verb_continuous))
+		message_verb_continuous = "[pick(I.attack_verb_continuous)]"
+	if(length(I.attack_verb_simple))
+		message_verb_simple = "[pick(I.attack_verb_simple)]"
```
```suggestion
	var/message_verb_continuous = length(I.attack_verb_continuous) ? "[pick(I.attack_verb_continuous)]" : "attacks"
	var/message_verb_simple = length(I.attack_verb_simple) ? "[pick(I.attack_verb_simple)]" : "attack"
```
Ternary operators for syntactic sugar.

Rohesie commented on code/modules/antagonists/devil/true_devil/_true_devil.dm:
```diff
@@ -123,8 +123,8 @@
 	var/message_verb = ""
-	if(I.attack_verb && length(I.attack_verb))
-		message_verb = "[pick(I.attack_verb)]"
+	if(I.attack_verb_continuous && length(I.attack_verb_continuous))
```
```suggestion
	if(length(I.attack_verb_continuous))
```
`length()` will return a falsey value without runtiming for falsey inputs

Rohesie commented on code/_onclick/item_attack.dm:
```diff
@@ -172,21 +172,26 @@
-	var/message_verb = "attacked"
-	if(length(I.attack_verb))
-		message_verb = "[pick(I.attack_verb)]"
+	var/message_verb_continuous = "attacks"
+	var/message_verb_simple = "attack"
+	if(length(I.attack_verb_continuous))
+		message_verb_continuous = "[pick(I.attack_verb_continuous)]"
+	if(length(I.attack_verb_simple))
+		message_verb_simple = "[pick(I.attack_verb_simple)]"
```
```suggestion
	var/message_verb_continuous = length(I.attack_verb_continuous) ? "[pick(I.attack_verb_continuous)]" : "attacks"
	var/message_verb_simple = length(I.attack_verb_simple) ? "[pick(I.attack_verb_simple)]" : "attack"
```

kingofkosmos committed:
Co-authored-by: Rohesie <rohesie@gmail.com>
Update code/_onclick/item_attack.dm
Update code/modules/antagonists/devil/true_devil/_true_devil.dm
Update code/modules/mob/living/carbon/carbon_defense.dm

kingofkosmos commented:
Thank you @Rohesie!
